xain_fl/grpc/participant.py

```diff
@@ -83,6 +83,8 @@ def end_training(
         history (obj:`History`): History metadata.
         Metrics (obj:`Metrics`): Metrics metadata.
     """
+    # TODO:
```

Comment:
Can you please create a ticket for that TODO and link it here

Reply:
Removed the TODO here.